scripts/tests/test_tv_suites.sh

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env bash
```

Comment:
I thought our testing strategy for this was to keep using yamls to describe the test but make it possible for the yaml to specify the app to run.

As a stopgap, because that would take some work, we could have a separate directory for the tv yamls and run the tv app as the backend for those tests using the same general `test_suites.sh` script but passing it args for where to look for yamls and which server app to run.

We shouldn't need to completely duplicate `scripts/tests/test_suites.sh` and we definitely shouldn't be hardcoding the specific test steps in the script; that does not scale out as we build up the tests.

Reply:
Updated. Added yaml files for almost all TV clusters. 

However I've still left the duplicate of `test_suites.sh` as I don't know how's it's run and I think I also don't have permission to edit those.

Anyhow, test_suites and test_tv_suites are pretty similar now and it should be easy to merge them into one file with arguments.

Reply:
> as I don't know how's it's run

It's run from `.github/workflows/tests.yaml`.  You should absolutely have permissions to edit that.

Reply:
Updated. Removed the `test_tv_suites.sh`

Now if you run test_suites.sh you should have the same experience as before, but if you run it with -a "app-name" argument, you can target the tests to be app specific.

I'm using match of `TV_*` to filter tv tests and `Test*` to filter the already existing tests.